experimenter/experimenter/experiments/models.py

```diff
@@ -1255,8 +1255,8 @@ def schemas_between_versions(
 
     @dataclass
     class VersionedSchemaRange:
-        # The versioned schemas in the requested range, or a single element list
-        # with an unversioned schema.
+        # The schemas that are supported in the requested version range, or a single
+        # element list with an unversioned schema.
```

Comment:
WDYT @brennie?

Reply:
There is no such thing as an "unsupported schema" because the model literally won't exist. I would leave it as is.